pkg/apis/eksctl.io/v1alpha5/iam.go

```diff
@@ -194,6 +194,15 @@ type PodIdentityAssociation struct {
 
 	// +optional
 	Tags map[string]string `json:"tags,omitempty"`
+
+	// +optional
+	// TargetRoleARN is the Amazon Resource Name (ARN) of the IAM role to be chained to the IAM role specified as RoleARN.
+	// This enables cross-account access where the RoleARN is in the same account as the cluster and TargetRoleARN is in a different account.
+	TargetRoleARN string `json:"targetRoleARN,omitempty"`
+
+	// +optional
+	// DisableSessionTags disables the tags that are automatically added to role session by Amazon EKS.
+	DisableSessionTags bool `json:"disableSessionTags,omitempty"`
```

Comment:
An optional boolean probably needs to be a pointer if we want to be able to differentiate yes/no/not provided

What if I have a pod identity with disable session tags=true and i update it to false? How do we differentiate explicit false vs not provided?

If this is only used for create, maybe that's ok since the default is false, either not providing the parameter in the create api request or explicitly setting false should be equivalent. But for update it's a different story.